.github/workflows/main.yml

```diff
@@ -2,7 +2,7 @@ name: Deploy Hugo site to Pages
 
 on:
   push:
-    branches: ["main", "cache-images"]
+    branches: ["main", "student-talk"]
```

Comment:
Wouldn't this deploy the student-talk branch? Is that what we want?

Reply:
Good question. I could have pruned that bit, but was being lazy. Basically the only place where I'm pushing student-talk branch was to my github, and this was to trigger the build there. I'll update this to a trigger in the commit message (eventually)

It wouldn't build over our production as new 'branches' have to additionally be configured in the github config (at least that has been my experience)